RELEASING.md

```diff
@@ -23,31 +23,43 @@ All patch releases should include only bug-fixes, and must avoid adding/modifyin
 
 In general, patch releases are only made for regressions, memory leaks and deadlocks.
 
-* Backport pull request(s) to the release branch
+* Backport pull request(s) to the release branch.
```

Comment:
Under `Preparing a new major or minor release` can you add some detail to what to add to CHANGELOG, maybe a small example? Notably want to document the practice of `VERSION - Unreleased`